requirements.txt

```diff
@@ -1,6 +1,5 @@
-django==1.7
+django
```

Comment:
And unbounded version of django is likely to cause us fits.  In fact, I'd like to get every dependency nailed down to an appropriate range, if not locked at a specific version.  That way anyone coming to the project gets the same versions of dependencies.